src/net_processing.cpp

```diff
@@ -175,6 +176,7 @@ static constexpr double MAX_ADDR_RATE_PER_SECOND{0.1};
 static constexpr size_t MAX_ADDR_PROCESSING_TOKEN_BUCKET{MAX_ADDR_TO_SEND};
 /** The compactblocks version we support. See BIP 152. */
 static constexpr uint64_t CMPCTBLOCKS_VERSION{2};
+static constexpr std::chrono::seconds TIME_OFFSET_WARN_THRESHOLD{30min};
```

Comment:
This is reduced from 70min to 30min, compared to `DEFAULT_MAX_TIME_ADJUSTMENT`. 70min avoids warning if you're buggy by an hour due to (you or your peers) getting daylight savings time wrong, which seems like a vaguely plausible thing to continue doing?